src/lib/github/updateExistingIssues.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * API function to add comments to existing bot created issues.
+ */
+
+import { Context } from "./types";
+import { Issue } from "./addIssuesToRepo";
+import { getBasicRepoProps } from "./utils";
+
+type Issues = Issue[];
+
+export default async function updateExistingIssues (context: Context, newIssue: Issue): Promise<Issues> {
+  const octokit = context.github;
+
+  const options = {
+    filter: "subscribed",
+    state: "open",
+    sort: "created",
+    order: "asc"
+  };
+
+  const results = await octokit.issues.getAll(options);
+
+  return results.issues.forEach( async (issue: Issue) => {
+  if (issue.title === newIssue.title ) {
+
+    const issueNumber = context.payload.issue.number;
+    const { owner, repo } = getBasicRepoProps (context);
+
+    const fields = {
+      owner,
+      repo,
+      issueNumber,
+      body: newIssue.body
+    };
+
+    return await octokit.issues.createComment(fields);
```

Comment:
`forEach` doesn't do anything with returned value i.e has no return value. Look up its documentation. So no need to return anything  here.